src/Avalonia.Controls/Converters/PlatformKeyGestureConverter.cs

```diff
@@ -57,7 +57,7 @@ public static string ToPlatformString(KeyGesture gesture)
             }
             else
             {
-                return gesture.ToString();
+                return ToString(gesture, "Cmd");
```

Comment:
OSX handling logic is in the previous branch and it's more complicated than using "Cmd"